cmd/ctl/pkg/create/certificaterequest/certificaterequest.go

```diff
@@ -225,10 +255,44 @@ func (o *Options) Run(args []string) error {
 	}
 	req, err = o.CMClient.CertmanagerV1alpha2().CertificateRequests(ns).Create(context.TODO(), req, metav1.CreateOptions{})
 	if err != nil {
-		return fmt.Errorf("error creating CertificateRequest: %v", err)
+		return fmt.Errorf("error creating CertificateRequest: %w", err)
 	}
 	fmt.Fprintf(o.Out, "CertificateRequest %s has been created in namespace %s\n", req.Name, req.Namespace)
 
+	if o.FetchCert {
+		fmt.Fprintf(o.Out, "CertificateRequest %v in namespace %v has not been signed yet. Wait until it is signed...\n",
+			req.Name, req.Namespace)
```

Comment:
These sort of messages should really go to Stderr, but I assume that we use Stdout elsewhere, so we shouldn't change here.

Reply:
I don't see this as an error but as a description of the (expected) state of the command.

Reply:
Yeah, I agree. However stderr isn't just for errors, it's for output that isn't the main output of the command. For instance imagine we wanted to support `--output-cert-file -` to output to stdout. That would either get mixed in with the logging, or we would have to disable the logging in that case. If we write the log to stderr then we can support things like that easily and the user can redirect the two streams as they like.

Reply:
I see, so by "these sort of messages" you also mean things like "Private key written to file fileName.key"?

Reply:
Yeah, these log messages. `status certificate` outputting the status to stdout makes sense.